packages/vendor-core/package.json

```diff
@@ -28,9 +28,10 @@
     "@patternfly/react-catalog-view-extension": "^4.10.2",
     "@patternfly/react-core": "~4.97.2",
     "@patternfly/react-icons": "~4.9.2",
-    "@patternfly/react-styles": "~4.8.2",
+    "@patternfly/react-styles": "^4.10.2",
```

Comment:
Is this update necessary?  We want have all pf4 packages from the same release. currently we are using version 2021.02 (2021-02-17) 
https://www.patternfly.org/2021.02/developer-resources/release-notes